content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -413,28 +413,43 @@ When you don't need to attach additional information to the relation, you can mo
 
 ## MongoDB
 
-In MongoDB, m-n-relations are represented by:
+In MongoDB, many-to-many relations are represented by:
 
-- Relation fields on both sides, include a `@relation` attribute on _each side_
-- A scalar list of referenced IDs on each side, including the `@db.ObjectId` attribute
+- a `@relation` attribute on _each side_, with mandatory `fields` and `references` arguments
+- a scalar list of referenced IDs on each side, including the `@db.ObjectId` attribute
 
-The following example demonstrate an m-n relation between posts and categories:
+The following example demonstrates a many-to-many relation between posts and categories:
 
 ```prisma
 model Post {
   id          String     @id @default(auto()) @map("_id") @db.ObjectId
   categoryIDs String[]   @db.ObjectId
-  categories  Category[] @relation(fields: [categoryIDs])
+  categories  Category[] @relation(fields: [categoryIDs], references: [id])
 }
 
 model Category {
   id      String   @id @default(auto()) @map("_id") @db.ObjectId
   postIDs String[] @db.ObjectId
-  posts   Post[]   @relation(fields: [postIDs])
+  posts   Post[]   @relation(fields: [postIDs], references: [id])
 }
 ```
 
-### Querying MongoDB m-n relations
+Prisma validates many-to-many relations in MongoDB with the following rules:
+
+- The fields on both sides of the relation must have a list type (in the example above, `categories` have a type of `Category[]` and `posts` have a type of `Post[]`)
+- The `@relation` attribute must define `fields` and `references` arguments
+- The `fields` argument must have only one scalar field defined, which must be of a list type
+- The `references` argument must have only one scalar field defined. This scalar field must exist on the referenced model and must be of the same type as the scalar field in the `fields` argument, but singular (no list)
+- The scalar field to which `references` points must have the `@id` attribute
+- No [referential actions](/concepts/components/prisma-schema/relations/referential-actions) are allowed
+
+The implicit many-to-many relations [used in relational databases](/concepts/components/prisma-schema/relations/many-to-many-relations#implicit-many-to-many-relations) are not supported on MongoDB.
```

Comment:
Want to check what this means... this is still a type of implicit relation, right? So does it mean that Mongo uses different syntax for its implicit relations?